Xamarin.Forms.Core/CornerRadiusTypeConverter.cs

```diff
@@ -58,7 +58,7 @@ public override string ConvertToInvariantString(object value)
 		{
 			if (!(value is CornerRadius cr))
 				throw new NotSupportedException();
-			return $"{cr.TopLeft}, {cr.TopRight}, {cr.BottomLeft}, {cr.BottomRight}";
+			return $"{cr.TopLeft.ToString(CultureInfo.InvariantCulture)}, {cr.TopRight.ToString(CultureInfo.InvariantCulture)}, {cr.BottomLeft.ToString(CultureInfo.InvariantCulture)}, {cr.BottomRight.ToString(CultureInfo.InvariantCulture)}";
```

Comment:
nit: instead of repeating CultureInfo.InvariantCulture multiple times consider using denser form
`string.Format(CultureInfo.InvariantCulture, "{0}, {1}, {2}, {3}", cr.TopLeft, cr.TopRight, cr.BottomLeft, cr.BottomRight);`

There are other places like this

Reply:
You could also do:

```csharp
FormattableString.Invariant($"{cr.TopLeft}, {cr.TopRight}, {cr.BottomLeft}, {cr.BottomRight}");
```

Reply:
@GalaxiaGuy nice tip. but we still need to support netstandard1.0 :(

Reply:
@etvorun I choose to keep named parameters instead of reverting to positional